packages/one-app-runner/README.md

```diff
@@ -0,0 +1,257 @@
+
+It works by pulling a Docker image for One App and mounting your module to it as a volume.
+
+**Note:** If you are making changes to One App at the same time you are making changes to your Module(s) then you will still need to [clone One App, serve your module to it, and start it yourself](https://github.com/americanexpress/one-app).
+
+## 🤹‍ Usage
+
+Note that [Docker](https://www.docker.com/community-edition#/download) is required and must be installed before **one-app-runner** can be used.
+
+### Quick start
+
+In any [Holocron Module](https://github.com/americanexpress/one-app/blob/master/docs/api/API.md#modules) directory:
+
+1. Install:
+```bash
+npm install --save-dev @americanexpress/one-app-runner
+```
+
+2. Add the `one-app-runner` command to your `npm` start script and a runner config to your module's
+`package.json`. See below for an example:
+
+##### package.json
+```json
+"scripts": {
+  "start": "one-app-runner"
+},
+"one-amex": {
+  "runner": {
+    "modules": ["."],
+    "rootModuleName": "frank-lloyd-root",
+    "moduleMapUrl": "https://example.com/cdn/module-map.json",
+    "dockerImage": "one-app-dev:5.0.0",
+    "parrotMiddleware": "./dev.middleware.js"
+  }
+}
+```
+
+Use the runner config to tell `one-app-runner` what module map and One App version to use, what your root module is, where your module is (current directory in this case), and optionally where your [Parrot mocks](https://github.com/americanexpress/one-app/blob/master/docs/recipes/Mocking-API-Calls.md) middleware file is located.
+
+##  Command options reference
+
+**one-app-runner** can be configured via command options or a configuration object in `package.json`:
+
+### docker-image [Required]
+
+Docker image to use for One App. It allows you to specify what Docker image and tag to pull down for One App. This can be an
+image you have built and tagged locally but is not published to a Docker registry. Whatever value is
+provided is passed down to `docker pull` and `docker run`.
+
+Sample usage:
+
+```bash
+npx @americanexpress/one-app-runner --docker-image one-app-dev:5.0.0 --module-map-url https://example.com/cdn/module-map.json
+```
+
+Or in `package.json`
+
+```json
+"one-amex": {
+  "runner": {
+    "dockerImage": "one-app-dev:5.0.0",
+    "rootModuleName": "frank-lloyd-root",
+    "moduleMapUrl": "https://example.com/cdn/module-map.json"
+  }
+}
+```
+
+### module-map-url [Required]
+
+Location of module map for One App to use to fetch modules. This option is required.
+
+Sample usage:
+
+```bash
+npx @americanexpress/one-app-runner --one-app-version 5.0.0 --module-map-url https://example.com/cdn/module-map.json
+```
+
+Or in `package.json`
+
+```json
+"one-amex": {
+  "runner": {
+    "moduleMapUrl": "https://example.com/cdn/module-map.json",
+    "dockerImage": "dockerhub.com/oneamex/one-app-dev:5.0.0",
```

Comment:
ooh this answers my question above. Would be better if specify that you can provide a link to the location of the  docker image

Reply:
I updated all the examples to include the full location of the image to make it more clear!